cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume.adoc

```diff
@@ -0,0 +1,15 @@
+[id="cnv-cloning-vm-disk-into-new-datavolume"]
+= Cloning a virtual machine disk into a new DataVolume
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-cloning-vm-disk-into-new-datavolume
+toc::[]
+
+You can clone the PersistentVolumeClaim (PVC) of a virtual machine disk into
+a new DataVolume by referencing the source PVC in your DataVolume configuration
+file.
```

Comment:
Is there a good reason why a user might want to do one of these over the other? They seem similar enough that some clarity about why you'd pick one over the other might be helpful in both assemblies, if you have it.

Reply:
Maybe some specific use cases could be added later, but I don't think I have that info yet. It boils down to deciding whether or not to immediately create a VM.

Reply:
In one example you might first create a VM and prepare it (by updating, configuring, installing software, etc).  Then you may clone the disk to a new DataVolume to serve as the basis for a VM template.  In this case you would not require (nor want) a VM associated with that disk.  At a later time, you could create multiple VMs by cloning that disk using the second method (DataVolume template inside a VM object).  Hopefully this clears it up a little.